tests/unit/test_digest_agent_mvp.py

```
@@ -0,0 +1,204 @@
+"""Unit tests for DigestAgentMVP."""
+
+import pytest
+from unittest.mock import AsyncMock, MagicMock, patch
```

Ruff (0.11.9)

`unittest.mock.patch` imported but unused

Remove unused import: `unittest.mock.patch`

(F401)